docs/tf_model_binary_input.md

```diff
@@ -55,12 +56,12 @@ docker run -d -p 9000:9000 -v ${PWD}/resnet_v2/models:/models openvino/model_ser
 ### Running the inference requests from the client
 
 ```Bash
-cd client/python/ovmsclient/samples
```

Comment:
Could you add a section to clone the model server repository and start from there?

Reply:
Done